manifests/g/Geniatech/TotalPlayer/6.48.02/Geniatech.TotalPlayer.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.TotalPlayer
+PackageVersion: 6.48.02
+Installers:
+- InstallerUrl: https://www.geniatech.eu/wp-content/uploads/2017/04/Geniatech_TotalPlayer_6.48.02.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Geniatech_TotalPlayer_6.48.02.exe
+  InstallerSha256: 7598F216F100216A3F323A980B42BFB4F699E42BD7436E5D97EA98D210711E3E

```

Comment:
```suggestion
  InstallerSha256: 7598F216F100216A3F323A980B42BFB4F699E42BD7436E5D97EA98D210711E3E
  InstallerSwitches:
    Silent: /s
```

Reply:
Testing the `/s` switch yields successful log entries but no shortcuts or other installed items. The switch `/S` has similar results, while other switches launch the interactive installer. I'm unsure which switch should be used here.